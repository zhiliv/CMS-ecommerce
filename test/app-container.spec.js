import { shallowMount } from '@vue/test-utils'
import Container from '@/components/app/container/container.vue'

describe('app-container.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(Container, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        breakpoint: 'sm, md, lg, xl, xxl',
        classes: 'simple',
        styles: 'background: white',
        id: 'indent'
      },
    })
  })
  it('Содержимое html для app-container ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
