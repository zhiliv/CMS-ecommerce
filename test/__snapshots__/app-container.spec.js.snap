// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-container.js Содержимое html для app-container  1`] = `
<div
  class="container container-sm container-md container-lg container-xl container-xxl simple"
  id="indent"
  style="background: white;"
/>
`;
